Extensions/pokes.js

```diff
@@ -40,6 +41,7 @@ XKit.extensions.pokes = {
 	run: function() {
 		if (!window.location.href.match(/www.tumblr.com/)) return;
 		this.running = true;
+		this.selector = (XKit.page.react ? "[data-id] article" : ".post_avatar") + ":not(.poked):not(.unpokable)";
```

Comment:
this would be clearer with an intermediate variable and string interpolation:
```suggestion
        const post_selector = XKit.page.react ? "[data-id] article" : ".post_avatar";
		this.selector = `${post_selector}:not(.poked):not(.unpokable)`;
```

Reply:
👍